src/components/Profile/components/PersonalInfoList/index.js

```diff
@@ -237,6 +237,27 @@ const PersonalInfoList = ({
       />
     );
 
+  /* TODO:  RESTRICTED FOR ALUM VIEW ONLY, LIKELY FINAL CODE BLOCK, COMMENTED FOR TESTING
+    const updateInfoButton = isAlumni ? (
+    <Grid container justifyContent="center">
+      <Link className="gc360_link" to="/update">
+        <Button variant="contained" className={styles.update_info_button}>
+          Update Information
+        </Button>
+      </Link>
+    </Grid>
+  ) : null;
+  */
+  const updateInfoButton = myProf ? (
+    <Grid container justifyContent="center">
+      <Link className="gc360_link" to="/myprofile/update">
+        <Button variant="contained" className={styles.update_info_button}>
+          Update Information
+        </Button>
+      </Link>
+    </Grid>
+  ) : null;
+
```

Comment:
Once everything else is done and tested, please clean this up so there's only one (uncommented) updateInfoButton that's only visible to alumni.